src/site/content/en/react/accessibility-auditing-react/index.md

```diff
@@ -138,7 +138,7 @@ unecessarily included in the final production bundle.
 Now when you run the application during development, issues are surfaced
 directly to the Chrome DevTools console.
 
-<img class="w-screenshot w-screenshot--filled" src="./react-axe-devtools.png" alt="React Axe in Chrome DevTools">
+<img src="./react-axe-devtools.png" alt="React Axe in Chrome DevTools">
```

Comment:
I've been adding the filled styles to all of our floating window screenshots because it was something Jiwoong had in his designs. Did you not like the look of the padded background?

Reply:
Ah didn't know they were intentional for the floating window screenshots as well. Definitely didn't mind it, but I thought it would look slightly better without since these screenshots already have a box-shadow to them.

**Before:**

<img width="981" alt="Screen Shot 2019-05-02 at 9 36 25 AM" src="https://user-images.githubusercontent.com/12476932/57091330-c74f6a00-6cbd-11e9-8e06-c4a9e16126c1.png">

**After:**

<img width="919" alt="Screen Shot 2019-05-02 at 9 36 57 AM" src="https://user-images.githubusercontent.com/12476932/57091364-dcc49400-6cbd-11e9-809c-f37630cac37c.png">

Reply:
But if they're part of the design then I'll bring em back. Makes sense to keep it consistent throughout the site 👍

Reply:
rad thank you. Yeah since it was in his comps I just started doing it for any of the floating window shots. +1 for consistency.